src/interfaces/tickets.ts

```diff
@@ -0,0 +1,44 @@
+import { SortOrder } from 'antd/lib/table/interface';
+
+export type SortedInfo = {
+  order: SortOrder;
+  columnKey: string;
+};
+
+export type Item = {
+  key: string;
+  avatar: string;
+  ticket: string;
+  name: string;
+  updated: string;
+  joinDate: string;
+  date: string;
+  time: string;
+  priority: string;
+  status: string;
+};
+
+export type Priority = { high: number; normal: number; low: number };
+
+export type Column = {
+  title?: string;
+  key?: string;
+  dataIndex?: string;
+  render?: (val: Item, val2?: any) => JSX.Element | null | undefined;
```

Comment:
```suggestion
  render?: (val: Item, val2?: any) => JSX.Element | null;
```
`undefined` is redundant here